### PR TITLE
Some typos

### PR DESCRIPTION
**Pull request by remybar**
Several typos splitted in several commits when a description is useful.

- - -
This change is [Reviewable](https://reviewable.io/reviews/cairo-book/cairo-book/508)

### CODE REVIEW

File: src/ch09-01-how-to-write-tests.md [hunk 1]
```diff
@@ -316,7 +316,7 @@ When you want to benchmark the gas usage of a specific operation, you can use th
 let initial = testing::get_available_gas();
 gas::withdraw_gas().unwrap();
     /// code we want to bench.
-(testing::get_available_gas() - x).print();
+(initial - testing::get_available_gas()).print();
```
Comment (enitrat) [inline src/ch09-01-how-to-write-tests.md:319]: would you mind using the new `println!` macro for this while we're at it?

**Review by enitrat — COMMENTED** [review]
💯

File: src/ch09-01-how-to-write-tests.md (hunk 1 above)
Comment (remybar) [reply src/ch09-01-how-to-write-tests.md]: yes, of course 👍

Comment (remybar) [reply src/ch09-01-how-to-write-tests.md]: it's done !

**Review by enitrat — APPROVED** [review]
thanks!!

**Pull request merged**